static/themes/SSF/style.css

```diff
@@ -46,3 +46,9 @@
     background-image: url(../../img/sunflower_spin_throbber.gif);
     background-repeat: no-repeat;
 }
+
+/* Comments */
+
+#comment-form {
+    width: 100%;
```

Comment:
Did you reduce the screen size and check ?
Maybe this could be 50 % ?
IMO 100% doesn't look good.